cmds/core/wget/wget_test.go

```diff
@@ -165,6 +159,22 @@ func TestWget(t *testing.T) {
 	}
 }
 
-func TestMain(m *testing.M) {
-	testutil.Run(m, main)
+func TestDefaultOutputPath(t *testing.T) {
+	u1, _ := url.Parse("http://localhost:8080/")
```

Comment:
Ignoring errors returned by `url.Parse` seems a bit dangerous.   Maybe it works today but someone may add an invalid url in the future and this may get missed.

Given that you have 3 different test cases here maybe it makes sense to change this to a table-driven test design and avoid having to duplicate the url parsing and error checking for each individual case?

Reply:
Actually we don't need full url to infer output path so I remove all url parsing, and this also allow to use table driven tests.